messageStructs.go

```diff
@@ -137,6 +137,60 @@ type (
 		Sender    User
 		Timestamp time.Time
 		Message   string
+		UUID      string
+	}
+
+	broadcast struct {
```

Comment:
is this struct needed? Not sure what the difference in this one and the `Broadcast` struct above is

Reply:
I guess it's not really necessary, yeah. I was just being a bit cautious: right now ```BROADCAST``` gets unmarshalled into a regular message, which doesn't really make sense since (in dgg at least) it doesn't have a user or a timestamp field attached. My thought process was that maybe in strims those fields do get used, so I kept them, not wanting to break anything. :smile: